precompiles/assets-erc20/src/tests.rs

```diff
@@ -2423,6 +2427,94 @@ fn burn_overflow() {
 		});
 }
 
+#[test]
+fn get_owner() {
+	ExtBuilder::default()
+		.with_balances(vec![(CryptoAlith.into(), 1000), (Bob.into(), 2500)])
+		.build()
+		.execute_with(|| {
+			assert_ok!(ForeignAssets::force_create(
+				RuntimeOrigin::root(),
+				0u128,
+				CryptoAlith.into(),
+				true,
+				1
+			));
+
+			precompiles()
+				.prepare_test(CryptoAlith, ForeignAssetId(0u128), ForeignPCall::owner {})
+				.expect_cost(0)
+				.expect_no_logs()
+				.execute_returns_encoded(Address(CryptoAlith.into()));
+		});
+}
+
+#[test]
+fn get_issuer() {
+	ExtBuilder::default()
+		.with_balances(vec![(CryptoAlith.into(), 1000), (Bob.into(), 2500)])
+		.build()
+		.execute_with(|| {
+			assert_ok!(ForeignAssets::force_create(
+				RuntimeOrigin::root(),
+				0u128,
+				CryptoAlith.into(),
+				true,
+				1
+			));
+
+			precompiles()
+				.prepare_test(CryptoAlith, ForeignAssetId(0u128), ForeignPCall::issuer {})
+				.expect_cost(0)
+				.expect_no_logs()
+				.execute_returns_encoded(Address(CryptoAlith.into()));
```

Comment:
Can you set an issuer different than the owner before to test it is returning the correct field?
This is applicable to all new tests.

Reply:
Perfect